CactusTutorial.ipynb

```diff
@@ -315,21 +321,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
This changes the python interpreter from python3 to python2 which should not be done (it *should* be harmless, but there is not guarantee that eg the ET tutorial server actually has python2 packages for matplotlib installed).

Myself I tend to save ipython notebooks as usual then add them to a commit using:
```sh
git add --patch CactusTutorial.ipynb
git commit
```
which lets me select just that parts of the file that I would like to add (which in this case would be just the changes till line 25 above and skip the python kernel settings and version / checksum information.

Reply:
Thanks for the advice! This should be fixed now. Could you please check?